### DISCUSSION TITLE
Discussion Topics for Monday August 2 Engineer Sync

### DISCUSSION THREAD

**Discussion by jimduff-usds** [CDCgov/prime-reportstream#1643] [Q&A]
@ronaldheft-usds
@RickHawesUSDS
@mry-usds
@mauricereeves
@Mr0grog
@joemeree
@carlosfelix2
@pablocaviglia
@tlosbo
@cbroome
@td-usds
@jorg3lopez
@brick-green-agile6
@ahay-agile6

**Comment by td-usds** [comment]
- Thinking in TEU (aka usage of Containers in day-to-day operations): [2021-07-19 - Dev Sync - TEUs.pdf](https://github.com/CDCgov/prime-reportstream/files/6842409/2021-07-19.-.Dev.Sync.-.TEUs.pdf)
- Update on enforcing [signed commits](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/signing-commits.md)

**Comment by jimduff-usds** [comment]
Re-using this Discussion for July 26 - post your topics!

**Reply by td-usds** [reply]
Thoughts about integrating gitleaks into our workflow (through git hooks and a scheduled CI/CD job) see also https://github.com/CDCgov/prime-reportstream/pull/1674 and https://github.com/CDCgov/prime-reportstream/issues/1645

**Reply by tlosbo** [reply]
Wanting to get some clarification around the CSV Uploader and how it ties into the messaging copy the `/api/reports` endpoint returns. Had some slack exchanges with Hayley and she mentioned Aurora will be working on the uploader.

**Comment by carlosfelix2** [comment]
OBX-22 looks incorrect in schema.  
Schema: 
name: test_result_report_date
    type: DATETIME
    natFlatFileField:
    hhsGuidanceField: Test report date
    hl7Field: OBX-22

HL7 Spec:
v2.6 and newer
 OBX-22	
CNE	Mood Code	
OBX-22 Reserved for harmonization with V2.6

**Comment by jimduff-usds** [comment]
Any topics for Aug 2 Tech meeting?
